documentation/source/library_reference.rst

```diff
@@ -346,6 +346,19 @@ The Scheduler
     :member-order: bysource
 
 
+The ``cocotb.simulator`` module
+-------------------------------
+
+This module is the interface to the GPI interface.
```

Comment:
Does the `i` in gpi mean interface? Perhaps my original wording was bad, and this should be:
```suggestion
This module is a python wrapper to libgpi.
```

Reply:
@ktbarrett, any opinions?

Reply:
Looks like a bit of RAS syndrome.

Reply:
Done.